config/500-controller.yaml

```diff
@@ -37,6 +37,10 @@ spec:
         env:
         - name: PUBSUB_RA_IMAGE
           value: github.com/GoogleCloudPlatform/cloud-run-events/cmd/pubsub/receive_adapter
+        - name: PUBSUB_PUBLISHER_IMAGE
```

Comment:
Where are these new images used, or is that in future PRs?

Reply:
future PR. they are in the stack...